README.md

```diff
@@ -5,11 +5,87 @@ This repository holds two things:
 1. [`data/transition-sites`](data/transition-sites) - Configuration for sites being redirected to GOV.UK. This configuration is automatically loaded into the [Transition app](https://github.com/alphagov/transition)
 2. [`tools/`](tools/) - Miscellaneous scripts for updating tna_timestamps, configuring root domain redirects and generating mappings.
 
-## Sites
+## Adding a new site
+
+### Prerequisites
+
+Before you start, you need to know:
+
+* the domain name being transitioned and any aliases
+* the organisation that owns the site (and any additional organisations that should have access)
+* the new homepage for the old site - often this is the organisation's page on GOV.UK, but sometimes it can be a different page on GOV.UK
+
+### Create the site yaml file
+
+Substitute each argument and then run the following rake task (you may need to run `bundle install` first):
+```
+  rake new_site[abbr,whitehall_slug,host]
+```
+
+``abbr`` should be of the form:
+```
+  <owning_organisation_abbreviation>_<abbreviated_site_name>
+```
+
+Example for the "SPIRE Export Control" site which is owned by the Department for Business, Innovation and Skills':
+```
+  bis_spire
+```
+
+The ``abbr`` is used in the URL for the site in the transition app. It is used as the primary key for a site and so shouldn't be changed once imported into the transition app, or a duplicate site will be created.
+
+``whitehall_slug`` is the GOV.UK slug of the organisation that owns the site. This determines several things:
+
+* where the site is found in the Transition app
+* who has access to edit and create mappings for the site
+* what organisation name and branding is used on pages served by Bouncer for URLs which aren't redirected
+
+Extra organisations can be added later.
+
+`host` will be one of the hostnames that the site has, for example:
+```
+  www.example.com
+```
+
+Aside: extra hostnames can be added later.
+
+### Adding extra config to the site
+
+The generated file only contains fields which are absolutely required.
+
+Check that the `tna_timestamp` field is set. The rake task will try to find the latest archive by scraping the National Archives site, but this doesn't always work. You can find a list of the timestamps the National Archives have by going to their site, for example: [here are the timestamps for this Cabinet Office site](http://webarchive.nationalarchives.gov.uk/*/http://download.cabinetoffice.gov.uk>)
+
+Some of the optional fields are frequently used:
+
+* `aliases` - Almost every site will aliases to cover the host name with/without `www.` (depending on which host was used in the ``host`` field)
+* `options` - used to specify when a site's URLs have significant querystring parameters.
```

Comment:
Nitpick: For consistency, could you remove the full stop here or add it to the end of other bulleted lines?